test/browser/redux/reducers/users.test.js

```diff
@@ -90,6 +90,19 @@ describe('users', () => {
       expect(initialState === users.allUsers(initialState, action)).to.be.false;
     });
 
+    it('should handle UPDATE_PROFILE_SUCCESS', () => {
+      const action = {
+        type: actionTypes.UPDATE_PROFILE_SUCCESS,
+        payload: { profile, userId: 'a1b2c3' }
+      };
+      expect(users.allUsers(undefined, action)).to.deep.equal({
+        a1b2c3: profile
+      });
+      let initialState = {};
+      // test to be sure not *mutating* state object but rather returning new!
```

Comment:
We have the object invariant test helper for this. Haven't used it much in the uploader, that's still a TODO, but just note that tests for mutation should *not* be written like this in general. Examples for how to use the object invariant test helper are in blip and in a POC here in misc.test.js